h2o-genmodel/build.gradle

```diff
@@ -23,4 +25,9 @@ javadoc {
 
 dependencies {
     testCompile "junit:junit:${junitVersion}"
+    compile "org.apache.commons:commons-compress:1.8.1"
+    compile "net.sf.opencsv:opencsv:2.3"
```

Comment:
First one is a zip file reader (maybe can be removed, maybe not)
The second is a CSV parser. (Maybe we can write our own, but there should be one)